checker/jdk/nullness/src/javax/xml/parsers/FactoryConfigurationError.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2000, 2005, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ */
+
+package javax.xml.parsers;
+
+import org.checkerframework.checker.nullness.qual.Nullable ;
+
+/**
+ * Thrown when a problem with configuration with the Parser Factories
+ * exists. This error will typically be thrown when the class of a
+ * parser factory specified in the system properties cannot be found
+ * or instantiated.
+ *
+ * @author <a href="mailto:Jeff.Suttor@Sun.com">Jeff Suttor</a>
+ * @version $Revision: 1.7 $, $Date: 2010-11-01 04:36:09 $
+ */
+
+public class FactoryConfigurationError extends Error {
+    private static final long serialVersionUID = -827108682472263355L;
+
+    /**
+     *<code>Exception</code> that represents the error.
+     */
+    private Exception exception;
+
+    /**
+     * Create a new <code>FactoryConfigurationError</code> with no
+     * detail mesage.
+     */
+	@SuppressWarnings("nullness")
+    public FactoryConfigurationError() {
+        super();
+        this.exception = null;
+    }
+
+    /**
+     * Create a new <code>FactoryConfigurationError</code> with
+     * the <code>String </code> specified as an error message.
+     *
+     * @param msg The error message for the exception.
+     */
+	@SuppressWarnings("nullness")
+    public FactoryConfigurationError(String msg) {
+        super(msg);
+        this.exception = null;
+    }
+
+
+    /**
+     * Create a new <code>FactoryConfigurationError</code> with a
+     * given <code>Exception</code> base cause of the error.
+     *
+     * @param e The exception to be encapsulated in a
+     * FactoryConfigurationError.
+     */
+
+    public FactoryConfigurationError(Exception e) {
```

Comment:
Why not permit the formal parameter to be null, by marking it as `@Nullable`?  That's a legal value, and it seems unnecessary to reject clients that pass `null`.

Reply:
@mernst  commit : d903ab9 makes all the changes as pointed out.
Thanks.